selfdrive/car/gm/carcontroller.py

```diff
@@ -107,13 +107,22 @@ def update(self, CC, CS):
       if self.CP.networkLocation == NetworkLocation.gateway and self.frame % self.params.ADAS_KEEPALIVE_STEP == 0:
         can_sends += gmcan.create_adas_keepalive(CanBus.POWERTRAIN)
 
+    elif self.CP.networkLocation == NetworkLocation.fwdCamera:
```

Comment:
an else seems more appropriate here. fwdCamera just happens to mean "not openpilot long" in this case
```suggestion
    else:
```